osu.Server.Queues.ScoreStatisticsProcessor/Stores/BeatmapStore.cs

```diff
@@ -26,7 +32,7 @@ namespace osu.Server.Queues.ScoreStatisticsProcessor.Stores
     /// </summary>
     public class BeatmapStore
     {
-        private static readonly bool use_realtime_difficulty_calculation = Environment.GetEnvironmentVariable("REALTIME_DIFFICULTY") != "0";
+        private static readonly bool prefer_realtime_difficulty_calculation = Environment.GetEnvironmentVariable("PREFER_REALTIME_DIFFICULTY") != "0";
```

Comment:
I preferred the old naming of this envvar because this had `ALWAYS` in the name which feels more correct. This saying "prefer" gives me an impression that realtime will _sometimes_ be used - but no, that's not what this is - realtime will _always_ be used regardless of this envvar, it's just a question of _for which mods_ (with the flag off all mods that don't have db attributes well be realtimed, and with the flag on, _everything_ will).

Reply:
Yeah I think I agree with this. I think I changed it initially because I read the naming in the `BeatmapStore` (`use_realtime_difficulty_calculation`) and found `use` misleading because with these changes it might be used even if this is set to `0`. I can probably rename the env var back to `ALWAYS_REALTIME_DIFFICULTY` and make sure the name in `BeatmapStore` reflects that same naming.

Reply:
`ALWAYS_USE_REALTIME_DIFFICULTY` if anything to appease the [grammar concern](https://github.com/ppy/osu-queue-score-statistics/pull/274#discussion_r1680417961).

I would have preferred that you wait for @peppy response to this review thread.

Reply:
Sure, sorry - I'll keep it on `ALWAYS_REALTIME_DIFFICULTY` for now to avoid dancing around more changes.

Reply:
`ALWAYS_USE_REALTIME_DIFFICULTY` is fine by me. just force push the change if you want.